web_ui/src/shared/components/media-item-annotations-preview/edit-annotations-button/edit-annotations-button.component.tsx

```diff
@@ -12,20 +12,10 @@ interface EditAnnotationsButtonProps {
 }
 
 export const EditAnnotationsButton = ({ datasetIdentifier, mediaItem }: EditAnnotationsButtonProps): JSX.Element => {
-    const navigate = useNavigateToAnnotatorRoute(datasetIdentifier);
-
-    const goToAnnotator = () => {
-        navigate({ ...datasetIdentifier, mediaItem });
-    };
+    const href = useAnnotatorRoutePath(datasetIdentifier)({ ...datasetIdentifier, mediaItem });
```

Comment:
np: we're duplicating args here. since the useCallback's callback get's the datasetIdentifier already, can we not get rid of `useAnnotatorRoutePath`s param and just use `const { workspaceId, projectId, organizationId } = projectIdentifier;` inside the callback? i.e. (comment above with asterisk). To go even further we could maybe get rid of the currying?